code/modules/power/apc.dm

```diff
@@ -823,7 +823,7 @@
 		var/datum/species/ethereal/E = H.dna.species
 		var/charge_limit = ETHEREAL_CHARGE_DANGEROUS - APC_POWER_GAIN
 		if((H.a_intent == INTENT_HARM) && (E.drain_time < world.time))
-			if(cell.charge <= (cell.maxcharge / 2)) // ethereals can't drain APCs under half charge, this is so that they are forced to look to alternative power sources if the station is running low
+			if(cell.charge <= (cell.maxcharge / 20)) // ethereals can't drain APCs under half charge, this is so that they are forced to look to alternative power sources if the station is running low
```

Comment:
Can you explain this change properly instead of just sliding it in

Reply:
I brought it up while talking about some elzu things in general Dev, and was told it could be lowered, I find it to be of a great annoyance when trying to charge
 https://discord.com/channels/837744059291533392/837744059291533395/988897412334637167